### PR TITLE
list-jira-tickets 0.0.1

### PR DESCRIPTION
**Pull request by a-mlc**
TagCheck

### What to do if the build fails?

At the moment contributors do not have access to the CI workflow triggered by StepLib PRs. In case of a failed build, we ask for your patience. Maintainers of Bitrise Steplib will sort it out for you or inform you if any further action is needed.

### New Pull Request Checklist

*Please mark the points which you did / accept.*

- [x] __I will not move an already shared step version's tag to another commit__
- [x] I read the [Step Development Guideline](https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md)
- [x] I have a test for my Step, which can be run with `bitrise run test` (in the step's repository)
- [x] I did run `bitrise run audit-this-step` (in the step's repository - note, if you don't have this workflow in your `bitrise.yml`, [you can copy it from the step template](https://github.com/bitrise-steplib/step-template/blob/master/bitrise.yml).)
- [x] I read and accept the [Abandoned Step policy](https://github.com/bitrise-io/bitrise-steplib#abandoned-step-policy)

**New Step**
Thank you for the new Step share! The CI check might will fail due to our extended validation engine. Nothing to worry about yet, we will get back to you shortly.

### CODE REVIEW

File: steps/list-jira-tickets/0.0.1/step.yml [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    description: |
+      Adding the URL path of your JIRA workspace will output the ticket links instead of just a list of IDs.
+    is_expand: true
+    is_required: false
+    summary: null
+    title: JIRA ticket's URL path
+  url_path: null
+- opts:
+    description: |
+      JIRA ticket's Prefix
```
Comment (mateherber) [inline steps/list-jira-tickets/0.0.1/step.yml:57]: Maybe a sample in the description would help.

**Review by mateherber — CHANGES_REQUESTED** [review]

**Comment by mateherber** [comment]
Hi @a-mlc!

Thank you very much for your contribution. I've added several remarks to the PR, please consider fixing them.

File: steps/list-jira-tickets/0.0.1/step.yml [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+  brew:
+  - name: git
+  apt_get:
+  - name: git
+is_requires_admin_user: true
+is_always_run: false
+is_skippable: false
+run_if: ""
+inputs:
+- opts:
```
[the remark this answers is not in the document]
Comment (lpusok) [reply steps/list-jira-tickets/0.0.1/step.yml:31]: As long the YML is valid, we can and should accept it. I have seen a similar format in other contributions that did not yet cause issues.

[the remark this answers is not in the document]
Comment (mateherber) [reply steps/list-jira-tickets/0.0.1/step.yml:31]: Great, then I'll resolve this comment.

**Pull request closed without merging**